### PR TITLE
There no estimates of goals on dahboard

### PR DESCRIPTION
**Pull request by Troff8**
## PR includes

- [x] Bug Fix

## Related issues

Resolve [#1540](https://github.com/taskany-inc/issues/issues/1540)
## QA Instructions, Screenshots, Recordings
Снимок экрана 2023-08-18 в 15 33 08

### CODE REVIEW

**Review by LamaEats — APPROVED** [review]

**Review by LamaEats — CHANGES_REQUESTED** [review]
Looks right, now we need make some steps

We need to implements right way to get an estimates.
First of all append partial query which join estimate to goal query to `goalDeepQuery` by default;
Next step for all places where is using `goalDeepQuery` variable append calculated field `_estimate` that be filled from `getEstimateListFormJoin`.

You may also need to fix access to the `estimate` value in place of use

**Review by LamaEats — APPROVED** [review]
lgtm

**Review by awinogradov — CHANGES_REQUESTED** [review]
Now we have two ways to get goal estimates. One in this PR and another [here](https://github.com/taskany-inc/issues/blob/114380b99f1bc0a27d8a10def6628b206f5543ac/trpc/router/goal.ts#L141) and [here](https://github.com/taskany-inc/issues/blob/114380b99f1bc0a27d8a10def6628b206f5543ac/trpc/router/project.ts#L250). I'm not it's correct and works in the same way.

**Review by 9teen90nine — APPROVED** [review]
Should be tested - many fields removed

File: src/components/GoalsPage/GoalsPage.tsx [hunk 1]
```diff
@@ -217,7 +217,7 @@ export const GoalsPage = ({ user, ssrTime, defaultPresetFallback }: ExternalPage
                             tags={g.tags}
                             priority={g.priority}
                             comments={g._count?.comments}
-                            estimate={g._estimate?.length ? g._estimate[g._estimate.length - 1] : undefined}
+                            estimate={g.estimate?.length ? g.estimate[0].estimate : undefined}
```
Comment (awinogradov) [inline src/components/GoalsPage/GoalsPage.tsx:220]: Why this is not `goal._lastEstimate` instead?

Comment (Troff8) [reply src/components/GoalsPage/GoalsPage.tsx]: changed to _lastEstimate

**Review by awinogradov — APPROVED** [review]

**Pull request merged**